mlir/lib/Dialect/Linalg/IR/LinalgOps.cpp

```diff
@@ -149,15 +154,41 @@ static void fillStructuredOpRegion(OpBuilder &opBuilder, Region &region,
   // iterator_types is an auto-generated method.
 }
 
+/// Helper to create a typical indexing map for MatmulOp. Returns a list of
+/// AffineMap.
+static SmallVector<AffineMap, 3>
```

Comment:
I'd strongly suggest dropping the `, 3` here and everywhere. That results in spooky copies if the caller does not use `SmallVector<AffineMap, 3>` and uses `SmallVector<AffineMap>` instead. Less confusion for a few extra stack elements is OK I think.

Reply:
Done